docs/reference/indices/delete-index-template.asciidoc

```diff
@@ -37,7 +37,7 @@ DELETE /_template/template_1
 
 Use the delete index template API to delete one or more index templates
 
-include::templates.asciidoc[tag=index-template-def]
+include::{es-repo-dir}/glossary.asciidoc[tag=index-template-def-short]
```

Comment:
This pulls in an incomplete sentence. Kind of reads oddly as-is:

![Screen Shot 2020-07-22 at 11 25 40 AM](https://user-images.githubusercontent.com/40268737/88195551-2b858300-cc0e-11ea-92b5-68200195fd18.JPG)

That short def also doesn't mention index aliases.

Reply:
Yeah, reusing the short defs doesn't work. Will fix.